src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -24,6 +28,11 @@ internal void InitializeLicense(string licenseText)
 
             if (string.IsNullOrWhiteSpace(licenseText))
             {
+                if (FirstTimeUser() && (Debugger.IsAttached && SystemInformation.UserInteractive))
+                {
+                    TrackFirstTimeUsageEvent();
+                    SetupFirstTimeRegistryKeys();
```

Comment:
if the intention is to always create the registry key and therefore not have additional attempts if the first HTTP request failed, I'm good with that. It just wasn't clear to me whether that was intentional or not 👍